src/buffer/buffer.rs

```diff
@@ -924,21 +923,17 @@ mod tests {
             .content
             .iter()
             .map(Cell::symbol)
-            .join("");
-        let actual_styles = small_one_line_buffer
-            .content
-            .iter()
-            .map(|c| c.fg)
-            .collect_vec();
+            .fold(String::new(), |acc, x| acc + x);
```

Comment:
I definitely prefer `join("")` here as it expresses the intent rather than the mechanic and is clearer.